06-building-a-program-start-to-finish/reflecting-on-the-learning-experience/Output/README.MD

```diff
@@ -0,0 +1,6 @@
+#### Output
+- **Using**: Medium
+- **Deliver**: a post reflecting on your learning experience while producing these outputs
+- **With**: *independently*
+
+[Reflecting on the learning experience](https://medium.com/@imisioluwa.akande/learning-in-andela-a-memorable-experience-that-is-worthwhile-21ce00f0ee78)
```

Comment:
Yes, I have gone through it again and modified the my text composition where necessary